inst/htmlwidgets/r3dmol.yaml

```diff
@@ -4,10 +4,10 @@ dependencies:
     src: htmlwidgets/lib/jquery-3.5.1
     script: jquery.min.js
   - name: r3dmol
-    version: 1.5.4
-    src: htmlwidgets/lib/3Dmol-1.5.4
+    version: 1.6.2
+    src: htmlwidgets/lib/3Dmol
     # script: 3Dmol.js            # Debug
-    # script: 3Dmol.min.js        # Production with jquery
+    # script: 3Dmol-min.js        # Production with jquery
     # script: 3Dmol-nojquery.js   # Debug without jquery
-    script: 3Dmol-nojquery.min.js # Production without jquery
+    script: 3Dmol-nojquery-min.js # Production without jquery
```

Comment:
Same here.

Reply:
The new 1.6.2 version of 3Dmol.js is built as `3Dmol-nojquery-min.js`. Same with `3Dmol-min.js

I assume they changed how how the `slurp` file runs. But the `.` breaks the package (because it points to a file that doesn't exist) as the newly built files have `-` instead.

The tests fail and nothing works if I change `-` back to `.` because the paths break.

Reply:
@BradyAJohnston Oh, thank you for the information, I didn't notice the change.
Merge it.